omero/developers/cli/extending.rst

```diff
@@ -3,8 +3,17 @@ Extensions
 
 Plugins can be written and put in the ``lib/python/omero/plugins``
 directory. On execution, all plugins in that directory are registered
-with the |CLI|. Alternatively, the ``--path`` argument can be used to
-point to other plugin files or directories.
+with the |CLI|. 
+
+Alternatively, plugins can be added to any directory ending with
+``omero/plugins``. If this directory is part of the ``PYTHONPATH`` the |CLI|
+will automatically include them.
+
+For testing purposes the ``--path`` argument can be used to point to other
```

Comment:
The `--path` argument to which command?  Does this need a cross-reference to the docs for the argument?